Relax the requirements on infomap, there seems to be nothing that actually
requires 1.0.6, see https://github.com/ulfaslak/infostop/issues/23

Author: Viktor Rehnberg
diff -ruN infostop.orig/requirements.txt infostop/requirements.txt
--- infostop.orig/requirements.txt	2023-05-29 11:49:21.167909914 +0200
+++ infostop/requirements.txt	2023-05-29 11:51:58.742054668 +0200
@@ -1,5 +1,5 @@
 pybind11>=2.4
-infomap==1.0.6
+infomap>=1.0.6,<3
 folium>=0.7.0
 numpy
 tqdm
diff -ruN infostop.orig/setup.py infostop/setup.py
--- infostop.orig/setup.py	2023-05-29 11:48:03.503374132 +0200
+++ infostop/setup.py	2023-05-29 11:49:07.575466118 +0200
@@ -114,7 +114,7 @@
     setup_requires=["pybind11>=2.4"],
     install_requires=[
         "pybind11>=2.4",
-        "infomap==1.0.6",
+        "infomap>=1.0.6,<3",
         "folium>=0.7.0",
         "numpy",
         "tqdm",
